### DISCUSSION TITLE
How to include NAT field with IP Address bulk import?

### DISCUSSION THREAD

**Discussion by spragger** [netbox-community/netbox#15979] [Help Wanted!]
I've searched everywhere and maybe I've missed it but there doesn't seem to be a way to include the NAT (Inside or Outside) field in a bulk CSV import.  Is this possible or is there some other way?

**Comment by motoridersd** [comment]
It seems this isn't possible? Did you ever figure out a way to do it?

**Comment by spragger** [comment]
Nope :/

**Comment by netravnen** [comment]
The Rest API?

https://demo.netbox.dev/api/ipam/ip-addresses/ (on your local instance `<netbox>/api/ipam/ip-addresses/`)

https://demo.netbox.dev/api/schema/swagger-ui/ (on your local instance `<netbox>/api/schema/swagger-ui/`)

The fields in the JSON body are there.

The CSV importer are indeed missing options.

The full feature set for importing data is the REST API way.

**Reply by netravnen** [reply]
Example, import the bulk data with the CSV importer.

use PATCH calls in the REST API to add missing data.

**Reply by motoridersd** [reply]
Ah nice! Thank you!
